spec/models/seller_extra_info_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe SellerExtraInfo, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Could you add a test:
    it { should belong_to(:seller) }

Reply:
Can I do that at this point even though I haven't migrated sellers to know about SellerExtraInfos?

I added this one line in and the test failed though that might totally be because I'm missing something else

Reply:
Ah you have to add subject { create :seller_extra_info } to use the SellerExtraInfo factory to generate a random object with an associated random seller